desktop/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
+# Aseprite Desktop Integration Module
+# Copyright (C) 2016  Gabriel Rauter
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
```

Comment:
About this Gabriel, I'll re-license this to MIT in a near future in case this code is moved to laf library: https://github.com/aseprite/laf